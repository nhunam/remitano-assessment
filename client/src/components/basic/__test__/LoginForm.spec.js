import LoginForm from "components/basic/Form/Login";
import React from "react";
import { shallow } from "enzyme";

describe("Login Form", () => {
  const onChangeMock = jest.fn();
  const wrapper = shallow(<LoginForm onChange={onChangeMock} />);

  it("should render properly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
