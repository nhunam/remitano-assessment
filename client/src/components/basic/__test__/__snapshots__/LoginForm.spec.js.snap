// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Form should render properly 1`] = `ShallowWrapper {}`;
